README.md

```diff
@@ -132,12 +132,42 @@ currently, there are verification tasks for the ILP32 (32-bit) and the LP64 (64-
 
 In order to obtain verification tasks from the programs and specifications in the repository,
 a simple task-definition mechanism is used.
-For each program, the repository contains a .yml file that specifies 
-  - the subject files (the parts that a program consists of),
-  - the results (for each specification of the program, the expected outcome is indicated), and
-  - the parameters to be used for verification.
+For each program, the repository contains a .yml file that specifies the following items:
+  - `format_version`: the version of the format (a version string)
+  - `input_files`: the subject program files or directories
+    (a list of file or directory names that a program consists of)
+  - `properties`: the properties that constitute the specification of the program,
+    each consisting of the following items:
+    - `property_file`: file that contains a property definition
+    - `expected_verdict`: the intended verification result (`true` or `false`)
```

Comment:
This property is also optional.

Reply:
I would say it is mandatory, not optional. The CI (in case we extend the CI checks) should complain if the verdict is missing for a given property.

Reply:
Even for coverage-Properties? Which verdict do you expect for them?

Reply:
Since you requested a new review, I'm pushing this conversation. I don't think it is resolved yet.

Reply:
You are right. There is an exception for coverage! Fixed.